src/parser/jobs/brd/DoTs.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @author Yumiya
+ */
+import React, {Fragment} from 'react'
+import Module from 'parser/core/Module'
+import STATUSES from 'data/STATUSES'
+import ACTIONS from 'data/ACTIONS'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {Rule, Requirement} from '../../core/modules/Checklist'
+
+const DROP_TOLERANCE = 1000
+
+export default class DoTs extends Module {
+	static handle = 'dots'
+	static dependencies = [
+		'checklist',
+		'util',
+	]
+
+	_dotComponent = new DotComponent(STATUSES.CAUSTIC_BITE.id, STATUSES.STORMBITE.id)
```

Comment:
Is this actually being used anywhere? I'm not seeing it unless I missed a side-effect.

Reply:
Not yet, I'll comment it out. I'm experimenting with it in another branch :eyes: